apps/cowswap-frontend/src/common/pure/IconSpinner/index.tsx

```diff
@@ -64,12 +64,16 @@ const Wrapper = styled.div<{ size: number; spinnerWidth: number; bgColor: string
   }
 `
 
+enum ColorVariables {
+  CONTAINER_BG_01 = '--cow-container-bg-01',
+}
```

Comment:
Yes, that's what I meant!
Sort of 😅 

This is the idea, but the suggestion is to put all of the variable (names?) in this enum.
Then you use the enum in the source css where they are declared, and when naming them individually.

Reply:
This change by itself is breaking the build.